spec.bs

```diff
@@ -1165,6 +1369,173 @@ Note that there is no `paymentRequestOrigin` field in
 {{CollectedClientAdditionalPaymentData}}, because the origin of the calling
 frame is already included in {{CollectedClientData}} of [[webauthn-3]].
 
+# Browser Bound Key Store - <dfn interface>BrowserBoundKeyStore</dfn> Internal Component # {#sctn-browser-bound-key-store}
+
+This component in the user agent manages platform-dependant cryptographic
+keypairs including their association to SPC credentials (i.e. passkeys). This
+section explains the |BrowserBoundKeyStore| and procedures for creating,
+binding, retrieving, and signing using the browser bound keypairs.
+
+The browser bound key store contains |browser_bound_map|, a map from
+[[=Credential ID=]] to a keypair identifier. The user agent stores
+this map on disk.
+
+## Creating a KeyPair Internal Method ## {#sctn-creating-a-keypair}
```

Comment:
Probably more for your spec mentor to help with, but I'm not sure that these are internal methods as defined in https://tc39.es/ecma262/#sec-object-internal-methods-and-internal-slots

(And I'm not sure if an internal component is a concept at all?)

Reply:
As long as they aren't attached to an interface, they have no impact on webpages. So it works just as is.

Often we refer to these as "steps to do XYZ", but thats not strictly required, I think it's just convention.

Reply:
Exactly, you basically just want an algorithm whose title is a definition with some steps, and you can call that algorithm from any other place in the spec, including actual web API methods. See https://wicg.github.io/fenced-frame/#revoke-network-for-a-partition-nonce and https://github.com/WICG/fenced-frame/blob/master/spec.bs#L2216-L2226 for example. Let me know if that helps.